### ISSUE TITLE
Add gcc address sanitizer

### ISSUE THREAD

**Issue by ruslo** [ruslo/polly#4]

**Comment by bhack** [comment]
:+1:

**Comment by ruslo** [comment]
@bhack you can use [reactions](https://help.github.com/articles/about-discussions-in-issues-and-pull-requests/#reacting-to-ideas-in-issues-and-pull-requests) to express +1

**Comment by bhack** [comment]
Yes I know but reactions are not available in the mobile view. I hope that Github will let to have that available also on mobile view.
